### PR TITLE
Contact form edited

### PR DESCRIPTION
**Pull request by GloireIrenge**
In this pull request we :

- reorient the submit button  of the contact form
- change the font size of the text inside the contact form

### CODE REVIEW

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @GloireIrenge, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] No linters errors :heavy_check_mark: 
- [ ] Good commit history :heavy_check_mark: 
- [ ] Descriptive PR :heavy_check_mark: 

## Required Changes ♻️

- [ ] As you had already implemented the contact section, I am not able to select your code to leave you comments so I will do it directly in my summary :grin: :grin:
- [ ] Please for your form the `action` value should be similar to `https://formspree.io/f/{form_id}` and the `method` value equals `POST` :upside_down_face: :upside_down_face:, please kindly fix it.
- [ ] Please the name input has `maxlength="30"` attribute, kindly add it :v: :v:
- [ ] Also message input has `maxlength="500"` attribute :grin: :grin: 
- [ ] All input fields have the `required` attribute to activate the validation on submit, please kindly add it to all your input field :grin: :grin: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -215,10 +215,10 @@ <h2>I'm always interested in hearing about new projects, so if you'd like to cha
           
               <div class="grid-item">
                 <div class="forms">
-                    <form action="#" method="GET">
-                        <input type="text" placeholder="Full name">
-                        <input type="email" placeholder="Email adress">
-                        <textarea placeholder="Enter text here"></textarea>
+                    <form action="#" method="POST">
+                        <input type="text" placeholder="Full name" maxlength="30" required>
```
Comment (HAMMAS-SALEEM) [inline index.html]: - [ ] Your form is not working. Please use [Formspree](https://formspree.io/html/) in your form. :+1: You see the instructions from this [link](https://formspree.io/html/)

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @GloireIrenge :wave:

### Highlights :1st_place_medal:

:heavy_check_mark: Following Linters 
:heavy_check_mark: Following GitHub Flow


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

# Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @GloireIrenge ✋



Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Professional Pull request ✔️
## Optional suggestions
- I would have used inline comments but files were changed and I have no access to the necessary files in order to insert inline comments.
- [ ] [Optional] You may please customize your readme more and remove the second author and the live-demo link since it is not linking anywhere. This will make your readme file more professional
- [ ] [Optional] You may as well use semantic tags in your HTML file. Please make the `section` container `main`, and then put the works in the `section` tag, the contact section in a `section`, the about me in a `section`, and also use the `footer` tag.
- [ ] [Optional] You may as well customize your logo with your name. This is because this is your portfolio, not anyone else's.
- [ ] [Optional] Please check the font family of your text across the whole project. It seems there is a slight difference that needs to be corrected.
- [ ] [Optional] Please check the spacing of the elements on your design. Compare them carefully with the Figma design provided. Check the social icons in the `footer` section, the black line below these icons, and the white space between the first page and the Recent works section
- [ ] [Optional] You may increase the width of your input sections in the form. This area seems smaller than the one provided in the Figma design
- [ ] [Optional] You may as well make the hamburger icon in your navbar a clickable link.

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**